src/frontend.md

```diff
@@ -421,6 +421,53 @@ export function fakeTicket() {}
    // no additional network request needed, use the first one
    ```
 
+4. Add query filters to query keys.
```

Comment:
not only filters, path parameters as well

Reply:
> btw, what about https://tanstack.com/query/v4/docs/react/eslint/eslint-plugin-query ?

good idea, added a task into our backlog: https://github.com/OsomePteLtd/principles/pull/254

Reply:
yeah, will consider to remove it after eslint rule covers all cases